doc/devel/min_dep_policy.rst

```diff
@@ -0,0 +1,38 @@
+.. _min_deps_policy:
+
+======================================
+Minimum Version Of dependencies Policy
```

Comment:
Not quite sure, what the capitalization policy really is, but everything capitalized except "dependencies" is a bit weird.

[devdocs](https://matplotlib.org/devel/documenting_mpl.html#section-name-formatting) says

> For everything but top-level chapters, use Upper lower for section titles

Don't know if this is a top-level chapter (and if so, what the policy for these were.

Reply:
That is indeed a typo...